storybook/story-components/prop-table.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
```

Comment:
Was there a requirement to make a table that shows props being used? The sole purpose of being able to run the Storybook in the simulator is purely being able to jump to stories and test components in isolation. The real estate on the simulator isn't large enough really to appreciate the Storybook UI. 

There was previously a way to have the Storybook UI in the browser and then choose stories to display in the simulator. I'm not sure if that's still a thing that can be done 🤔 

But, it may be a better workflow.

Reply:
The reason we have a custom solution is because we can't use the standard addon-docs package with `storybook/react-native`. addon-docs includes a prop-table that I've found useful in the past so I added it. If it's not needed I can remove it.

I have no idea how it would be possible to link from the browser to the simulator. Do you have a link?

Reply:
Erm. This is how Storybook initially worked with `react-native` scripts.

You would run `Storybook` that would run in a browser instance. You'd choose a story from the browser and it would display in the simulator. This was the default behavior. But, I'm not sure how you activate that behavior any more or if it's available. I'll need to explore that.